src/codegen/table.cpp

```diff
@@ -75,7 +75,9 @@ void Table::DoGenerateScan(CodeGen &codegen, llvm::Value *table_ptr,
   // Get start and end index from multi thread context.
   llvm::Value *thread_id = codegen.Const64(0);
   llvm::Value *num_tile_groups = GetTileGroupCount(codegen, table_ptr);
-  MultiThreadContext multi_thread_context;
+
+  uint32_t num_threads = 3;
```

Comment:
does this mean for each query, there will be a maximum of 3 threads executing it?